sections/appendix/models.tex

```diff
@@ -0,0 +1,56 @@
+		\textbf{Author-Doc Model} & \textbf{888036.4} & \textbf{0.543} & \textbf{0.574} \\
+		\textbf{Category-Doc Model} & \textbf{896045.7} &\textbf{ 0.530} & \textbf{0.575} \\
+	\end{tabular}
+	\label{tab:_extra_metric_results}
+\end{table}
+
+As \autoref{tab:_extra_metric_results} shows, the perplexity score for the two models, are extremely high, since we are multiplying three vectors instead of two.
+When looking at the topics, that the two models produce and comparing them, we can see some suttle difference which might indicate the influence of the metadata.
+\begin{table}
+		\centering
+	\caption{Top 10 words for two topics within the extension models}
+	\begin{tabular}{l|c|c|c|c|c|c|c|c|c|c}
+		Model & 1 & 2 & 3 & 4 & 5 & 6 & 7 & 8 & 9 & 10 \\
+		\midrule
+		Author-Doc & wozniacki & vandt & open & sæt & turneringen & caroline & hobro & runde & nummer & arige \\
+		Category-Doc & vandt & wozniacki & nummer & runde & open & sæt & turneringen & par & slag & dansk \\
+		\midrule
+		Author-Doc & eu & brexit & britiske & storbritannien & aftale & may & parlamentet & sagde & london & premierminister \\
+		Category-Doc & eu & brexit & britiske & storbritannien & parlamentet & may & aftale & europa & london & johnson \\
+		\midrule
+		Author-Doc & natur & dyr & landbrug & naturen & landmænd & skov & hektar & vand & lille & danmarks \\
+		Category-Doc & naturen & natur & du & dansk & hvordan & maske & omradet & landbrug & penge & kystsikring \\
+	\end{tabular}
+	\label{tab:_extra_metric_results}
+\end{table}
+We have taken 3 topic pairs, which seemed to be about the same topics and compared them between the models.
+The first topic pair in \autoref{tab:_extra_metric_results}, is about tennis, since some of the words are talking about Caroline Wozniacki, who is a professional danish tennis player and "turneringen" the tournament.
+Specifically, "caroline" is not in the top 10 of the Category-Doc model which might indicate that the authors have written about "caroline wozniacki" before in other contexts.
+The word "slag" (hit) is also used within tennis, which might indicate that the Category metadata helps bring sport words higher up in the ranks.
+Otherwise the two topics are pretty similar in the words they use.
+
+Looking at the second pair of topics, which is about the EU and Great Britain, we can see that they are very similar.
+The 8th word for the Author-topic model, is "sagde" (said), which is not a very informative word, regarding the topic, but a author might use these kinds of words many times during an article.
+Other than that, the topics are very similar, when looking at the 10 most probable words.
+
+The third pair of topics, is about nature and farming.
+These two topics are not as similar as the other two pairs, we have looked at but they have two different viewpoints on this topic.
+The Author-Doc model describes words concerning farming, since two of the words used, are "landbrug" (farming) and "landmand" (farmer). 

```

Comment:
```suggestion
The Author-Doc model describes words concerning agriculture since two of the words used are "landbrug" (agriculture) and "landmand" (farmer). 
```